src/app/debug/table/table.component.html

```diff
@@ -79,7 +79,7 @@ <h4 class="progress-report-timout">[One or more reports are in progress for more
 
       @switch (getAmountOfReportsSelected()) {
         @case (1) {
-          <button data-cy-debug="openReportTab" class="btn btn-info btn-table" title="Open In New Tab" (click)="openReportInTab()">
+          <button class="btn btn-info btn-table" title="Open In New Tab" (click)="openReportInTab()">
```

Comment:
Why did you remove the `data-cy` tag here?

Reply:
Woops. I don't know how that happened actually. I added it back.